modules/system/classes/UpdateManager.php

```diff
@@ -582,7 +582,7 @@ public function extractPlugin($name, $hash)
         $fileCode = $name . $hash;
         $filePath = $this->getFilePath($fileCode);
 
-        if (!Zip::extract($filePath, $this->baseDirectory . '/plugins/')) {
+        if (!Zip::extract($filePath, App::pluginsPath())) {
```

Comment:
```suggestion
        if (!Zip::extract($filePath, plugins_path())) {
```

Reply:
I am a little not biased towards helpers, given that they can be removed in some of the releases. Using the method of the Application object is due to the presence and use of this method in the core as a whole. I could be wrong. Based on your offer, I see no reason to refuse.

Reply:
I suggest using the helpers because they are used elsewhere in the core while App::pluginsPath() isn't used elsewhere anywhere. It is extremely unlikely that October will ever remove helpers (in fact we're re-adding Laravel's helpers that they removed in our upgrade to L6) since we're big on maintaining compatibility.

Reply:
> I suggest using the helpers because they are used elsewhere in the core while App::pluginsPath() isn't used elsewhere anywhere. It is extremely unlikely that October will ever remove helpers (in fact we're re-adding Laravel's helpers that they removed in our upgrade to L6) since we're big on maintaining compatibility.

Yes OK! I do not see any contradictions with this option.